src/util/nextThought.js

```diff
@@ -31,56 +31,58 @@ import {
  * Returns true if the context view is activated and has more than one context
  * @returns {boolean}
  */
-const isValidContextView = context =>
-  context.length &&
-  isContextViewActive(context) &&
-  getContexts(head(context)).length > 1
+const isValidContextView = path =>
+  path.length &&
+  isContextViewActive(pathToContext(path)) &&
+  getContexts(head(path).value).length > 1
```

Comment:
Technically `head` should only be called on `Path` types, `Contexts` are meant to be unordered. Since the current implementation of `Context` is an array, it ends up working on both. I haven't been consistently using `head` in this way myself so this is not that important right now.

Reply:
Do you think we can create a util function that returns the last item from a context?

Reply:
No, the concept of "first" and "last" assumes that the list is ordered, and a `Context` should be considered unordered. It is implemented as array for simplicity. A more advanced implementation would probably represent it as a `Set` and have some serialization logic for persistence.

As mentioned, the codebase unfortunately does not respect the unordered property of `Contexts` in many cases.

`Paths` and `Children` are ordered.